tuna/parsing.py

```diff
@@ -377,15 +377,22 @@ def arg_valid(arg, val):
   return True
 
 
-def parse_driver_line(line):
-  """return network parameters from driver invocation"""
-  fds = {}
+def parse_line(line):
+  """parse the lines from MIopenDriver"""
   line = line.strip()
   start = line.find('MIOpenDriver')
   if start == -1:
     raise ValueError(f"Invalid driver commmand line: '{line}'")
 
   line = line[start:]
+
+  return line
+
+
+def parse_driver_line(line):
```

Comment:
This function appears to be unused. Maybe delete this instead of moving parse_line function here?

Reply:
For the parse line, its used in the driver base

Reply:
Referring to parse_driver_line which is defined in parsing, but never used anywhere.

Reply:
This is a bigger issue, and i think its beyond the scope of this PR. parse_driver_line in parsing is used in parsing.py but the functions that call it are not called anywhere. I will file an issue to remove this now dead code (since removal of Grafana functionality).